From 7331a4cc0853722b4c3addf1927a2797f39f5de2 Mon Sep 17 00:00:00 2001
From: Peng Fan <Peng.Fan@freescale.com>
Date: Mon, 1 Jun 2015 16:09:32 +0800
Subject: [PATCH] MLK-11011 imx: mx6ul-14x14-evk update ddr version to 1.1
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

updated DDR Script of 6UL EVK Board to avoid a calibration error
when using “DDR_Stress_Tester_V1.04.exe”.

Updated items:
	[Modified]  setmem /32 0x020E027C = 0x00000008
	[Modified]  setmem /32 0x020E0280 = 0x00000038
	[Modified]  setmem /32 0x021B080C = 0x00070007
	[Added   ]  setmem /32 0x021B0858 = 0x00000F00

The script versions of EVK board and Validation Board from the following link:
http://compass.freescale.net/livelink/livelink?func=ll&objId=234191407&obj
Action=browse&viewType=1

Test Results:
Tested on two boards, both passed overnight memtester stress test.

Signed-off-by: Peng Fan <Peng.Fan@freescale.com>
---
 board/freescale/mx6ul_14x14_evk/imximage.cfg | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/board/freescale/mx6ul_14x14_evk/imximage.cfg b/board/freescale/mx6ul_14x14_evk/imximage.cfg
index 2af4d86..8bbb1fd 100644
--- a/board/freescale/mx6ul_14x14_evk/imximage.cfg
+++ b/board/freescale/mx6ul_14x14_evk/imximage.cfg
@@ -62,7 +62,7 @@ DATA 4 0x020c4084 0xffffffff
 
 DATA 4 0x020E04B4 0x000C0000
 DATA 4 0x020E04AC 0x00000000
-DATA 4 0x020E027C 0x00000030
+DATA 4 0x020E027C 0x00000008
 DATA 4 0x020E0250 0x00000030
 DATA 4 0x020E024C 0x00000030
 DATA 4 0x020E0490 0x00000030
@@ -72,17 +72,16 @@ DATA 4 0x020E0260 0x00000030
 DATA 4 0x020E0264 0x00000030
 DATA 4 0x020E04A0 0x00000030
 DATA 4 0x020E0494 0x00020000
-DATA 4 0x020E0280 0x00000030
+DATA 4 0x020E0280 0x00000038
 DATA 4 0x020E0284 0x00000030
 DATA 4 0x020E04B0 0x00020000
 DATA 4 0x020E0498 0x00000030
 DATA 4 0x020E04A4 0x00000030
 DATA 4 0x020E0244 0x00000030
 DATA 4 0x020E0248 0x00000030
-
 DATA 4 0x021B001C 0x00008000
 DATA 4 0x021B0800 0xA1390003
-DATA 4 0x021B080C 0x00000000
+DATA 4 0x021B080C 0x00070007
 DATA 4 0x021B083C 0x41490145
 DATA 4 0x021B0848 0x40404546
 DATA 4 0x021B0850 0x4040524D
@@ -91,6 +90,7 @@ DATA 4 0x021B0820 0x33333333
 DATA 4 0x021B082C 0xf3333333
 DATA 4 0x021B0830 0xf3333333
 DATA 4 0x021B08C0 0x00922012
+DATA 4 0x021B0858 0x00000F00
 DATA 4 0x021B08b8 0x00000800
 DATA 4 0x021B0004 0x0002002D
 DATA 4 0x021B0008 0x1B333000
-- 
1.8.0

